### PR TITLE
document tilt improvements

### PR DESCRIPTION
**Pull request by saweber**
Document improvements contained in PRs:
https://github.com/opencost/opencost/pull/2428
https://github.com/opencost/opencost-helm-chart/pull/163

### CODE REVIEW

File: docs/installation/development.md [hunk 1]
```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
...
+7. In a terminal, run `tilt up` at the root of the OpenCost repository. See [Tilt Command Options below for more options.](#options)
+8. Follow instructions in the terminal to open the Tilt UI in browser.
+9. As you update code, `tilt_config.json`, or `tilt-values.yaml`, Tilt will automatically watch relevant source code and recompile, build, deploy, and run tests.
+10. `tilt down` will tear down resources created, otherwise they will continue to run indefinitely.
+11. `kind delete` cluster will delete the local cluster, if desired.
+
+## Tilt Command Options
+
+### via `tilt_config.json`
+The best way to configure OpenCost to use tilt is with a `tilt_config.json` file in the root of the OpenCost repository. The file must live there; a symlink is thebest alternative if you wish to store it elsewhere. This file is automatically reloaded in tilt when updated. Omitted values will use defaults.
```
Comment (mattray) [inline docs/installation/development.md:37]: "thebest" should be "the best"

Comment (mattray) [reply docs/installation/development.md:37]: Also sometimes you use "Tilt" and other times "tilt", probably stick with "Tilt" everywhere.

File: docs/installation/development.md [hunk 2]
```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
...
+  Optional. Defaults to an empty string.
+  This is the path to the JSON file to use for the `cloud-integration` secret. [See configuration documentation here.](../configuration)
+- `delve-continue`
+  Value: `true`(default) or `false`
+  Optional, defaults to true. Determines if the `--continue` flag is used with delve. By default, this will let applications start without a debugger attached. `false` will wait for a debugger to attach before starting.
+- `docker-repo` 
+  Value: Docker Hub username or repository, defaults to an empty string.
+  Required for a remote cluster, optional for local. 
+  Optional. This is the CPU architecture of the desired node(s) on the cluster.
+  The value is prepended to the docker image names (the -t flag in a Docker build command). [See Docker Documentation](https://docs.docker.com/engine/reference/commandline/build/#tag).
```
Comment (mattray) [inline docs/installation/development.md:66]: You kinda mix "Docker" and "docker", I think "docker" is just the CLI command and everything else should be capitalized.

File: docs/installation/development.md [hunk 3]
```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
...
+- `helm-values`
+  Optional. Defaults to `./tilt-values.yaml`
+  This is the path to the values.yaml file that will be used for the OpenCost helm chart.
```
Comment (mattray) [inline docs/installation/development.md:69]: Having both "tilt-values.yaml" and "values.yaml" feels a little confusing, maybe leave off the ".yaml" and just call it the "values file that will be used". Also "helm" should be capitalized.

File: docs/installation/development.md [hunk 4]
```diff
@@ -20,19 +20,75 @@ Tilt is a great way to run OpenCost on a local or remote Kubernetes environment.
...
+  This is the path to the JSON file to use for the `service-key` secret. [See configuration documentation.](../configuration)
+- `port-costmodel`
+  Optional. Defaults to 9003.
+  This is the port forwarded to localhost for the costmodel (back end).
+  Note: In `tilt_config.json`, this is a string and must be in quotes, since Tilt options must be a bool or a string.
+- `port-debug`
+  Optional. Defaults to 40000.
+  This is the port forwarded to localhost for delve.
+  Note: In `tilt_config.json`, this is a string and must be in quotes, since Tilt options must be a bool or a string.
+- `port-prometheus`
```
Comment (mattray) [inline docs/installation/development.md:82]: I think Prometheus is listening on 80 inside the OpenCost cluster, does this forward 80 to 9080?

**Review by mattray — CHANGES_REQUESTED** [review]
Mostly picky stuff, I'll approve and test

File: docs/installation/development.md (hunk 4 above)
Comment (saweber) [reply docs/installation/development.md:82]: Correct. The only configurable port through tilt is the 'local' side.

**Comment by saweber** [comment]
Issues addressed. I appreciate the detailed review!

**Review by mattray — APPROVED** [review]

**Pull request merged**